src/Core/SonarLintNotifications/NotificationSetting.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Core.SonarLintNotifications
+{
+    public interface INotificationSetting
+    {
+        string Key { get; }
+        bool DoNotShow { get; }
```

Comment:
Is the notification setting likely to have more properties that just `DoNotShow`?
I guess it might, but I can't think of any examples right now.

Reply:
I also thought about it but couldn't really find a good example.